grails-app/conf/application.yml

```diff
@@ -20,6 +20,11 @@ environments:
             url: jdbc:mysql://localhost/streama
             username: root
             password:
+    development-no-mysql:
+            dataSource:
+                dbCreate: update
+                driverClassName:  'org.h2.Driver'
```

Comment:
It's still a choice for developers. If I don't push it, it's a risk having conflicts if you or someone else commit something here.